<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignatureForm</class>
 <widget class="QWidget" name="SignatureForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signature</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="emailLabel">
     <property name="text">
      <string>&amp;Email:</string>
     </property>
     <property name="buddy">
      <cstring>emailEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="emailEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="timeLabel">
     <property name="text">
      <string>&amp;Time:</string>
     </property>
     <property name="buddy">
      <cstring>timeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateTimeEdit" name="timeEdit">
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string notr="true">yyyy-MM-dd HH:mm:ss (t)</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nowButton">
       <property name="text">
        <string>Now</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>nowButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
